### ISSUE TITLE
Cable Tiers - 0.51 - Disk Manipulators

### ISSUE THREAD

**Issue by GMGKenny** [starforcraft/Cable-Tiers#19]
Hey there! I just wanted to let you know that from what I've seen/tested it appears that the higher tiers of disk manipulator don't have any additional speed. They are actually slower than the regular one because there aren't any upgrade slots. Not sure if this is intended, but I thought I would mention it just in case. Hope you're doing great! Thanks for your time :)

**Comment by starforcraft** [comment]
Yeah I know this issue but idk why it happens because if you test the mod out without any other mods except RS it works.

**Comment by GMGKenny** [comment]
Oh okay, sorry. I didn't realize you already knew about this. Thanks for getting back to me though! :)

**Comment by starforcraft** [comment]
Are you playing on a Server btw?

**Comment by GMGKenny** [comment]
Yes it's on a server.

**Comment by starforcraft** [comment]
Hmm seems like its a server issue
